README.md

```diff
@@ -1,73 +1,45 @@
-# Node.js (React, Express) Project Starter ✅
+# Cool Husky Games
 
-A simple template repository with a React frontend and Express backend for quickly starting new full-stack projects.
+## Uploading your game files to the website
 
-# Notes for deploying to AWS
-The <script> tags in the head of the `public/index.html` and `public/404.html` files need to be removed, as they are GitHub pages hack specific.
+1. Run pygbag to compile your pygame-ce file and assets for the website. When done, the output folder should look like [this](https://drive.google.com/drive/folders/15yVl406Rvm_zXHZ0qWUeB_uFUsEN8hYQ?usp=drive_link).
 
-The router also assumes a `basepath` of `/cool-husky-games` in the `createRouter` function call in `src/index.js`. This works because of how GitHub Pages is set up, but on AWS the website will be
-served from the root path and doesn't have the `/cool-husky-games` subpath, so that specific line will need to be removed as well.
+2. Find the compiled files at `/[game-name]/build/web`. In the pong example, this would be `index.html`, `favicon.png`, `pong.apk`, and `pong.html`.
```

Comment:
Will they have to make/rename to get index.html, `favicon.png`, `pong.apk`, and `pong.html` files themselves or will running pygbag do this automatically for them?

Reply:
Pygbag as mentioned in step one will do it